setup.py

```diff
@@ -46,9 +46,7 @@
 extras['all'] = extras['dev'] + extras['examples']  # + extras['docs']
 
 # These packages shall be installed only on GPU machines
-PACKAGES_GPU_ONLY = (
-    'horovod',
-)
+PACKAGES_GPU_ONLY = ('horovod')
```

Comment:
what the heck?
that's not equivalent
one is a tuple, the other is a string!
if yapf did this automatically, that's highly concerning

Reply:
heh, that was my bad, I remove the, manually :/

Reply:
ok, it looks like it's not yapf, just a manual edit